Pages/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,10 @@
-﻿<!DOCTYPE html>
+﻿@{ 
+    var darkMode = string.Equals(Context.Request.Query["theme"], "dark")
```

Comment:
nice

Reply:
![Nathan Fillian trying to tell someone not to disclose a secret by waving his hand and putting a finger up to his mouth in a "shh" gesture](https://www.bing.com/th?id=OGC.7f5f4d999e79148c572197e301eb5eb2&pid=1.7&rurl=https%3a%2f%2fmedia.giphy.com%2fmedia%2fKUOPgSNoKVcuQ%2fgiphy.gif&ehk=ABUoncZaZLGBWk%2bMFJoUnQ)

Dark Theme Easy Mode